src/renderer/components/SplittedChatListAndView.js

```diff
@@ -27,13 +27,22 @@ class SplittedChatListAndView extends React.Component {
   }
 
   onChatClick (chat) {
-    console.log('xx', chat)
+    console.log('xx', chat.id)
     this.setState({ chatId: chat.id })
   }
 
+  getChat () {
+    const { deltachat } = this.props
+    const { chatId } = this.state
+    const index = deltachat.chats.findIndex(chat => {
+      return chat.id === chatId
+    })
+    return deltachat.chats[index]
+  }
+
   render () {
     const { deltachat } = this.props
-    const { chat } = this.state
+    const chat = this.getChat()
```

Comment:
When rendering we _must_ get the chat object from `this.props.deltachat.chats` array.